threatbus/data.py

```diff
@@ -42,6 +44,12 @@ class Operation(Enum):
     REMOVE = "REMOVE"
 
 
+@unique
+class MatchType(Enum):
```

Comment:
Let this inherit from `str` to enable JSON serialization:
```python
class MatchType(str, Enum):
     LIVE = "LIVE"
     RETRO = "RETRO"
```
Otherwise I got:
```
Object of type MatchType is not JSON serializable
```